index.html

```diff
@@ -12,6 +12,13 @@
 <body>
     <main>    
         <h1>Finance Tracker</h1>
+        <!--Form goes here-->
+
```

Comment:
Can we please add this to the body
    <select id="list-dropdown" name="list">
    </select>